### PR TITLE
New post: Using Java, Amazon Corretto, Ukulele and Fast Fourier Transform to detect musical notes and command IoT devices

### PR DESCRIPTION
**Pull request by vsenger**
# Content Pull Request Checklist

Please check to make sure your content submission meets the following requirements by adding an `x` between the brackets:

- [X] The title of this pull request reflects the title of my content.
- [X] I have reviewed the [Content Review Checklist](/content/blob/main/CONTENT_REVIEW_CHECKLIST.md).
- [X] This pull request contains one piece of content (post or page).

**Important: if you are submitting a PR to update existing content, please ensure you [sync'ed your fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork) first before you created your branch.**

Is this pull request linked to a Content Proposal or Idea Suggestion Issue? If so, please link to it below using the number or full link.

```text
Resolves #5

or

Content for https://github.com/build-on-aws/content/issues/5
```

**By submitting this pull request, I confirm I own this content, have permission to use each image included in this content, and that it will be released under the [CC BY-SA 4.0 SA License](/LICENSE).**

### CODE REVIEW

**Comment by bketelsen** [comment]
@vsenger I fixed all the problems in the markdown/frontmatter, but the image sizes for a few of the files are too large and can't be merged. Can you resize the images mentioned in the reviewbot's comments above please?

**Review by cherbk — APPROVED** [review]

File: tutorials/fourier-iot/index.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+9. Copy the endpoint address to some place. We will need this information to configure our Processing.org connection to AWS IoT Core.
+![AWS IoT Core Screenshot](./images/iot07.png)
+
+Now we have a representation of our notes Thing in the AWS Cloud and also the security keys to communicate using TLS 1.2.
+
+## Detecting Notes Sketch
+
+Now that we have installed Processing.org, Minim library, and already have our Thing created in AWS IoT Core, let's start to explore the Sketch we did for detecting and converting notes to IoT / MQTT Messages!
+
+Open Processing.org and copy and paste this code (don't worry - we'll explain the details later):
```
Comment (damonk10) [inline tutorials/fourier-iot/index.md:136]: You promised to explain the details later, but I don't see where you did that. Can you explain what this code is actually doing?

File: tutorials/fourier-iot/index.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+## Let's Build!
+
+So let's explore how the FFT can be put to work. The architecture of our solution is simple, and for now we are using the AWS IoT Core service to manage messages that we will be able to react to later:
+
+![implemented archicture](./images/architecture1.png)
+
+1. The first component is your laptop with microphone running Processing.org with a notes detection script.
+2. Once it detects a note, it will send the note name to the "control/sound" IoT topic.
+3. From AWS IoT Core we can forward the message to other topics. For example, if the note is a G, we could turn on the smart bulb in a lamp by sending a MQTT message.
+4. We can also trigger other services like AWS DynamoDB for storing the message data without one line of coding, or we could call an AWS Lambda function: sound oriented functions!
```
Comment (damonk10) [inline tutorials/fourier-iot/index.md:44]: I don't understand the end of line 44. Is it just something you're emphasizing? Like, "Imagine sound-oriented functions!" If so, maybe edit to something like this:

"... or we could call an AWS Lambda function (yes, we could literally build sound-triggered functions!)."

File: tutorials/fourier-iot/index.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+
+Then you can check the AWS Console:
+
+![AWS IoT Core Screenshot](./images/iot10.png)
+
+[Check out the live-demo video here!](https://youtu.be/0oavSqRD4wc)
+
+## Conclusion
+
+Now you can imagine infinite possibilities: our team already did home automation integration to turn on and off lamps using Ukulele; we made it rain inside Minecraft based on notes, and we also changed Roblox game behavior. Now it's up to you to build on your own stuff using this magical algo: Fast Fourier Transform. Have fun!
```
Comment (damonk10) [inline tutorials/fourier-iot/index.md:303]: Can you add a few more links to related content for curious readers?

File: tutorials/fourier-iot/index.md (hunk 2 above)
Comment (vsenger) [reply tutorials/fourier-iot/index.md:44]: thanks for the suggestion! done.

File: tutorials/fourier-iot/index.md (hunk 1 above)
Comment (vsenger) [reply tutorials/fourier-iot/index.md]: I change the idea to explain the code inside comments... Any chance for someone to test the step-by-step in a "new" computer?

Comment (vsenger) [reply tutorials/fourier-iot/index.md]: David, how I can embed the video like Ricardo did in his tutorial??

**Review by cherbk — APPROVED** [review]

File: tutorials/fourier-iot/index.md (hunk 1 above)
Comment (cherbk) [reply tutorials/fourier-iot/index.md]: @vsenger here are the instructions for embedding video in posts 
https://github.com/build-on-aws/content/blob/main/FAQ.md#how-do-i-embed-youtube-videos-in-my-post

**Review by cherbk — APPROVED** [review]

**Pull request merged**